apps/services/license-api/src/app/modules/license/license.module.ts

```diff
@@ -50,8 +54,20 @@ import { DisabilityLicenseApiClientConfig } from './clients/disabilityLicense/di
       useFactory: (
         disabilityClient: DisabilityLicenseClientService,
         firearmClient: FirearmLicenseApiClientService,
-      ) => async (type: LicenseId): Promise<GenericLicenseClient | null> => {
+        drivingLicenseConfig: ConfigType<typeof DrivingLicenseApiClientConfig>,
+        xRoadConfig: ConfigType<typeof XRoadConfig>,
+      ) => async (
+        type: LicenseId,
+        cacheManager: CacheManager,
+      ): Promise<GenericLicenseClient | null> => {
```

Comment:
The `clientFactory` in `LicenseService` does not define the `cacheManager` parameter is this used anywhere?

Reply:
the old pkpass client